dockerfile_images/basic/superbuild-icubhead-robometry/Dockerfile

```diff
@@ -19,9 +27,18 @@ ARG INSTALL_DIR
 ARG release
 ARG sbtag
 ARG METADATA_FILE
+ARG ROBOTOLOGY_INITIALIZATION_FILE
+ARG USERNAME
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+USER $USERNAME
+
+COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+COPY setup.sh ${ROBOTOLOGY_INITIALIZATION_FILE}
+RUN sudo chown -R icub: /usr/local/bin/entrypoint.sh
+RUN sudo chown -R icub: /usr/local/bin/setup_robotology_tdd.sh
+
 # Some other definitions
 ARG CMAKE_GENERATOR="Unix Makefiles"
 ARG BUILD_TYPE=Release
```

Comment:
Hi @MSECode ,
this is only a question: why did you move the `ARG sbtag`and ``ARG release` on top, while you left BUILD_TYPE here?